source/agora/common/Set.d

```diff
@@ -236,6 +263,28 @@ unittest
     old_set.each!(value => assert(value in new_set));
 }
 
+/// toString and fromString test for Set!int
+unittest
+{    
+    auto old_set = Set!uint.from([1, 3, 5, 7, 9]);
+    auto str = old_set.toString();
```

Comment:
The actual value should be tested, because those will pass as long as they are symmetric.

Reply:
Added a check to test the actual value.